scripts/reorder_json_keys.py

```diff
@@ -0,0 +1,35 @@
+import json
+import os
+
+from recipe_scrapers._utils import get_abstract_methods
+
+KEYS = get_abstract_methods()
+
+
+def reorder_json_keys(file_path):
+    with open(file_path, encoding="utf-8") as file:
+        data = json.load(file)
+
+    reordered_data = {key: data[key] for key in KEYS if key in data}
+
+    # Serialize both original and reordered data for comparison
+    original_content = json.dumps(data, indent=2, ensure_ascii=False)
+    reordered_content = json.dumps(reordered_data, indent=2, ensure_ascii=False)
+
+    if original_content != reordered_content:
```

Comment:
These three operations -- re-ordering the data, serializing it to JSON, and comparison of the serialized outputs -- could all be relatively computationally expensive.

Both `json.loads` and `dict` objects in recent Python versions generally retain the ordering of the data that they load, so I think we can skip a lot of this computation. 

Let's instead try:

```suggestion
    # Rewrite if the loaded data does not use the currently-preferred order
    if list(data.keys()) != KEYS:
```